### PR TITLE
Replace SFGB with MAPS

### PR DESCRIPTION
**Pull request by benlovell**
Various changes to enact the switch from the Single Financial Guidance Body to
the Money and Pensions Service.

### CODE REVIEW

**Comment by benlovell** [comment]
[As a customer, I need to know that Pension Wise is provided by the Money & Pensions Service](https://trello.com/c/otaYUyXY/941-as-a-customer-i-need-to-know-that-pension-wise-is-provided-by-the-money-pensions-service)

File: config/locales/cy.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 cy:
   language_name: Cymraeg
 
+  layouts:
+    base:
+      provided_by: Pension Wise is provided by
```
Comment (benlovell) [inline config/locales/cy.yml]: @arwelowen Could you please translate this text: 'Pension Wise is provided by'? Thanks.

**Comment by arwelowen** [comment]
The translation you need is – ‘Darperir Pension Wise gan’

Arwel Owen
Welsh Language Unit | Department for Work and Pensions | Work Services Directorate | Wales Group| Out stationed at Pwllheli JCP | Lower Cardiff Road | Pwllheli | Gwynedd | LL53 5BY | Tel. 01758 632601| www.dwp.gov.uk<http://www.dwp.gov.uk/>
Uned yr Iaith Gymraeg | Adran Gwaith a Phensiynau | Cyfarwyddiaeth Gwasanaethau Gwaith | Grŵp  Cymru | Yn Gweithio allan o CBG Pwllheli | Lower Cardiff Road | Pwllheli | Gwynedd | LL53 5BY | Ffôn 01758 632 601 | www.dwp.gov.uk/cymraeg<http://www.dwp.gov.uk/cymraeg>
Please consider the environment before printing/Ystyriwch yr amgylchedd cyn argraffu

**Comment by benlovell** [comment]
@arwelowen Thanks!

**Pull request merged**